tuf/tuf.go

```diff
@@ -326,16 +326,16 @@ func delegationUpdateVisitor(roleName string, addKeys data.KeyList, removeKeys,
 			}
 		}
 		// We didn't find the role earlier, so create it only if we have keys to add
+		// that are also sufficient to meet the threshold.
```

Comment:
Is this comment still true?  It seems like we're creating the role even if there are no keys.